yggdrasilffi/src/lib.rs

```diff
@@ -8,7 +8,12 @@ use std::{
 use libc::c_void;
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use unleash_types::{client_features::ClientFeatures, client_metrics::MetricBucket};
-use unleash_yggdrasil::{Context, EngineState, EvalWarning, ExtendedVariantDef, ToggleDefinition};
+use unleash_yggdrasil::{
+    Context, EngineState, EvalWarning, ExtendedVariantDef, ToggleDefinition, CORE_VERSION,
+};
+
+static CORE_VERSION_CSTRING: std::sync::LazyLock<CString> =
```

Comment:
I think the `expect` here is okay since this should only cause problems if this uses an escape character to inject a nul byte and this always a semver string which can't have escape characters.

For the Unrusted:

CStrings are heap allocated so we can't just make constants out of them. So we wrap this in a LazyLock, which sets the value the first time and thereafter just retrieves it. A singleton, if you will

For the Rusted:

Look ma, no `once_cell`!